examples/with-mobx/components/Clock.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { inject, observer } from 'mobx-react'
+
+const Clock = inject('store')(observer(({store}) => {
```

Comment:
Can't we do this injecting part in the place where we use it?
If not, try to expose the actual clock component.
The idea is, we need to have a state logic independent UI component.